tailwind.config.js

```diff
@@ -11,6 +11,132 @@ module.exports = {
     ],
   },
   theme: {
+    colors: {
+      transparent: 'transparent',
+      current: 'currentColor',
+      white: '#ffffff',
+      black: {
+        700: '#000000',
+        600: '#000000',
+        500: '#000000',
+        400: '#141414',
+        300: '#292929',
+        200: '#3D3D3D',
+        100: '#525252',
+      },
+      red: {
+        900: '#351008',
+        800: '#7C2513',
+        700: '#9F2F19',
+        600: '#C23A1E',
+        500: '#DE4527',
+        400: '#E3674F',
+        300: '#E98672',
+        200: '#F1B3A7',
+        100: '#F7D2CA',
+        50: '#FCF0ED',
+      },
+      green: {
+        900: '#000000',
+        800: '#040B09',
+        700: '#1A392F',
+        600: '#316654',
+        500: '#209678',
+        400: '#59B595',
+        300: '#6AD0AD',
+        200: '#7CD8BB',
+        100: '#91DFC8',
+        50: '#9EE3CF',
+      },
+      orange: {
+        900: '#4D2E0D',
+        800: '#814D1A',
+        700: '#B56D27',
+        600: '#E98C35',
+        500: '#FBA239',
+        400: '#F2B76F',
+        300: '#F5C992',
+        200: '#F8DBB6',
+        100: '#FBEDDB',
+        50: '#FFF7ED',
+      },
+      grey: {
+        900: '#000000',
+        800: '#000000',
+        700: '#1A1A1A',
+        600: '#363636',
+        500: '#525252',
+        400: '#666666',
+        300: '#7B7B7B',
+        200: '#8F8F8F',
+        100: '#A4A4A4',
+        50: '#AEAEAE',
+      },
+      blue: {
+        900: '#0A181A',
+        800: '#1F4147',
+        700: '#336B74',
+        600: '#4994A2',
+        500: '#2EBFD2',
+        400: '#71C7D6',
+        300: '#89D2DE',
+        200: '#A3DCE6',
+        100: '#BEE7EE',
+        50: '#CDECF2',
+      },
+      zink: {
+        900: '#030303',
+        800: '#1F1F1F',
+        700: '#3B3B3B',
+        600: '#575757',
+        500: '#737373',
+        400: '#878787',
+        300: '#9C9C9C',
+        200: '#B0B0B0',
+        100: '#C5C5C5',
+        50: '#CFCFCF',
+      },
+      neutral: {
+        900: '#5C5C5C',
+        800: '#787878',
+        700: '#949494',
+        600: '#B0B0B0',
+        500: '#CCCCCC',
+        400: '#E0E0E0',
+        300: '#F5F5F5',
+        200: '#FFFFFF',
+        100: '#FFFFFF',
+        50: '#FFFFFF',
+      },
+      slate: {
+        900: '#31343A',
+        800: '#4A4F59',
+        700: '#949494',
+        600: '#7E8695',
+        500: '#9CA3AF',
+        400: '#B4B8C0',
+        300: '#CACDD2',
+        200: '#E0E2E6',
+        100: '#F6F7F8',
+        50: '#FFFFFF',
+      },
+      indigo: {
```

Comment:
Indigo is added for the old design, once we change the main page, the color to be removed